package/base-files/files/lib/upgrade/nand.sh

```diff
@@ -129,14 +129,26 @@ nand_upgrade_prepare_ubi() {
 
 	local ubidev="$( nand_find_ubi "$CI_UBIPART" )"
 	if [ ! "$ubidev" ]; then
-		ubiattach -m "$mtdnum"
+		if [ ! -z $CU_UBI_VID_HEADER_OFFSET ]; then
```

Comment:
You have don't set this variable below (commented out), so do you still need these changes?